package.json

```diff
@@ -24,10 +24,11 @@
   "dependencies": {
     "@types/node-fetch": "^2.5.10",
     "form-data": "^4.0.0",
-    "node-fetch": "^2.6.1"
+    "node-fetch": "^2.6.1",
+    "rimraf": "^3.0.2"
```

Comment:
Consumers of `freesound-client` don't need `rimraf`. To prevent them from installing it you can add it to devDependencies:

https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file

Reply:
Ah, I see. That makes sense. I believe cbb4a17 addresses this.

Thank you for the explanation!